script.js

```diff
@@ -0,0 +1,129 @@
+//DOM Elements
+let userScore = 0;
+let computerScore = 0;
+let round = 1;
+const paperBtn = document.querySelector(".Paper");
+const rockBtn = document.querySelector(".Rock");
+const scissorsBtn = document.querySelector(".Scissors");
+const userScore_span = document.getElementById("user-score");
+const computerScore_span = document.getElementById("computer-score");
+const round_span = document.getElementById("round");
+const remark_p = document.querySelector(".remark");
+const userChoice_p = document.querySelector(".userChoice");
+const compChoice_p = document.querySelector(".compChoice");
+const clickPaper = paperBtn.addEventListener("click", game);
+const clickRock = rockBtn.addEventListener("click", game);
+const clickScissors = scissorsBtn.addEventListener("click", game);
+
+//Create a Function (getPlayerChoice) to Grab user inputs as choice of paper, rock or scissors
```

Comment:
- I think it would nice to remove all comments from source code before merging to main.

Reply:
Thank you sir for the review I apologize for my late response

Reply:
All corrections n suggestions will be implemented asap